doc/source/tech_note/Introduction/CLM50_Tech_Note_Introduction.rst

```diff
@@ -524,11 +524,17 @@ light capture, carboxylation, and respiration are co-limiting.
 
 CLM5 applies a fixed allocation scheme for woody vegetation.  The decision to use a fixed allocation scheme in CLM5, rather than a dynamic NPP-based allocation scheme, as was used in CLM4 and CLM4.5, was driven by the fact that observations indicate that biomass saturates with increasing productivity, in contrast to the behavior in CLM4 and CLM4.5 where biomass continuously increases with increasing productivity (:ref:`Negron-Juarez et al., 2015<NegronJuarezetal2015>`).  Soil carbon decomposition processes are unchanged in CLM5, but a new metric for apparent soil carbon turnover times (:ref:`Koven et al., 2017 <Kovenetal2017>`) suggested parameter changes that produce a weak intrinsic depth limitation on soil carbon turnover rates (rather than the strong depth limitaiton in CLM4.5) and that the thresholds for soil moisture limitation on soil carbon turnover rates in dry soils should be set at a wetter soil moisture level than that used in CLM4.5.
 
-Representation of human management of the land (agriculture, wood harvest) is augmented in several ways. The CLM4.5 crop model is extended to operate globally through the addition of rice and sugarcane as well as tropical varieties of corn and soybean :ref:`(Badger and Dirmeyer, 2015<BadgerandDirmeyer2015>` and :ref:`Levis et al., 2016)<Levisetal2016>`.  These crop types are added to the existing temperate corn, temperature soybean, spring wheat, and cotton crop types.
-Fertilization rates and irrigation equipped area updated annually based on crop type and geographic region through an input dataset.  The irrigation trigger is updated.  Additional minor changes include crop phenological triggers that
+Representation of human management of the land (agriculture, wood harvest) is augmented in several ways. 
+The CLM4.5 crop model is extended to operate globally through the addition of rice, sugarcane,  
+tropical varieties of corn and soybean :ref:`(Badger and Dirmeyer, 2015<BadgerandDirmeyer2015>` and :ref:`Levis et al., 2016)<Levisetal2016>`, 
+and perennial bioenergy crops :ref:`(Cheng et al., 2019)<Chengetal2019>`.
+These crop types are added to the existing temperate corn, temperature soybean, spring wheat, and cotton crop types.
```

Comment:
```suggestion
These crop types are added to the existing temperate corn, temperate soybean, spring wheat, and cotton crop types.
```

Reply:
Another misspelling of temperate related to soy -- seems a common theme! Thanks for helping to correct this mistake, which is from prior to your work